### ISSUE TITLE
Please provide disk images for releases

### ISSUE THREAD

**Issue by NetwideRogue** [klange/toaruos#156]
I like the OS - I've been playing around with it using the live ISO for a while. But the live environment is obviously inflexible with regards to file storage, and my 8 GB of RAM don't seem to be enough to build the OS from source. I would really appreciate disk images on the GitHub releases.

**Comment by klange** [comment]
You shouldn't have any trouble building with 8GB of RAM.

That aside, distributing full disk images is problematic. The FS drivers are still very flaky, so I don't want to give users a reason to push them too hard. Beyond that, when I do get around to making sure they're stable, I plan on deploying an installer with the CD, so you could build your own disk image from that. The package manager is also not set up to work with full filesystems, so disk image releases would not be able to reliably install packages. By the time these issues are resolved, the installer should be available, so there's very little possibility that I'll ever provide disk image releases again.

**Comment by RogueAI42** [comment]
I think anybody sane would know that the system's not production ready yet, so they shouldn't be upset if something doesn't work. But obviously it's up to you.

I actually forgot the details of the build failure. Last time I tried to build it, the scripts still used "make -j", and so I was getting "Out of memory: Kill process or sacrifice child" building gcc. I limited that to make -j6 and encountered a completely different problem. I filed a new issue about it - #157.
